OpenRA.Game/Platform.cs

```diff
@@ -108,6 +108,13 @@ public static string ResolvePath(string path)
 			if (path.StartsWith("./") || path.StartsWith(".\\"))
 				path = GameDir + path.Substring(2);
 
+			if (path.Contains("@"))
```

Comment:
This should use `:` to denote `mod:path` for consistency with our package-specific filenames and our other yaml definitions.  See my explanation in IRC for how to implement this.